govcd/query_metadata.go

```diff
@@ -212,6 +215,7 @@ func (client *Client) cumulativeQueryWithHeaders(queryType string, params, notEn
 		types.QtResourcePool,
 		types.QtNetworkPool,
 		types.QtProviderVdcStorageProfile,
+		types.QtOrg,
```

Comment:
Since you have added some of the pieces needed to run Org queries through cumulative query, you should also add this to `addResults`

```go
	case types.QtOrg:
		cumulativeResults.Results.OrgRecord = append(cumulativeResults.Results.OrgRecord, newResults.Results.OrgRecord...)
		size = len(newResults.Results.OrgRecord)
```